src/app/functions/server/breachResolution.ts

```diff
@@ -228,24 +228,4 @@ function getRecommendationFromResolution(resolution: any, args: any) {
   return { header, body, priority };
 }
 
-/**
- * Take breach DataTypes array from HIBP and filter based on BreachDataTypes enums above
- *
- * @param originalDataTypes breach DataTypes array from HIBP
- * @returns filtered breach data types
- */
-// Old untyped code, adding type defitions now isn't worth the effort:
-/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-function filterBreachDataTypes(originalDataTypes: any[]) {
-  const relevantDataTypes = Object.values(BreachDataTypes);
-  // This function predates proper use of TypeScript, so we don't have a better
-  // return type than `any` yet:
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-  return originalDataTypes.filter((d) => relevantDataTypes.includes(d));
-}
-
-export {
-  BreachDataTypes,
-  appendBreachResolutionChecklist,
-  filterBreachDataTypes,
-};
```

Comment:
This function is not being used anywhere.

Reply:
![giphy](https://github.com/mozilla/blurts-server/assets/4251/af0c42af-f8fc-4309-a81f-948de2f7e0c4)